packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/review/eventgrid.api.md

```diff
@@ -50,8 +50,8 @@ export class EventGridClient {
     constructor(endpointParam: string, credential: KeyCredential, options?: EventGridClientOptions);
     acknowledgeCloudEvents(topicName: string, eventSubscriptionName: string, lockTokens: AcknowledgeOptions, options?: AcknowledgeCloudEventsOptionalParams): Promise<AcknowledgeResult>;
     readonly pipeline: Pipeline;
-    publishCloudEvent(topicName: string, event: CloudEvent, options?: PublishCloudEventOptionalParams): Promise<Record<string, any>>;
-    publishCloudEvents(topicName: string, events: CloudEvent[], options?: PublishCloudEventsOptionalParams): Promise<Record<string, any>>;
+    publishCloudEvent(topicName: string, event: CloudEvent, options?: PublishCloudEventOptionalParams): Promise<PublishResult>;
```

Comment:
The current generation should be correct.

Reply:
not sure i doubt this would be a bug in geteffectivemodel or somewhere not us, becuase the getAllModels didn't return it yet.

Reply:
I suspect it is because we treat anonymous empty model and named empty model differently? I vaguely remember there's a discussion about whether anonymous empty model should be an open dict ?

Reply:
yeah, i guess so and this is also template argument and this may be the impact also. but i can't reproduce now
https://cadlplayground.z22.web.core.windows.net/?c=aW1wb3J0ICJAdHlwZXNwZWMvaHR0cCI7Cgp1c2luZyBUeXBlU3BlYy5IdHRwOwpAc2VydmljZSh7CiAgdGl0bGU6ICJXaWRnZXQgU8YbIiwKfSkKbmFtZXNwYWNlIERlbW%2FHGzsKCm1vZGVsIEZvbyB7fQpvcCB0ZXN0KCk6IHtAYm9keSBfOsQcO307&e=%40typespec%2Fopenapi3&options=%7B%22linterRuleSet%22%3A%7B%22extends%22%3A%5B%22%40typespec%2Fhttp%2Fall%22%5D%7D%7D